superset/assets/javascripts/explorev2/components/ControlHeader.jsx

```diff
@@ -64,6 +65,7 @@ export default function ControlHeader({
               {' '}
             </span>
           }
+          <span>{leftNode}</span>
```

Comment:
should this be inside the <ControlLabel /> component? what is the use case for the left node? we probably also want to wrap in a conditional.
```
{leftNode && 
  ...
}
```